content/en/providers/check-in/sp/_index.md

```diff
@@ -1177,3 +1264,34 @@ the selected authentication mechanism as follows:
 
 - Username/password credentials → Low
 - X.509 certification → Substantial
+
+### GOCDB Roles
+
+#### Background
+
+GOCDB roles are encoded in entitlements
+(`eduPersonEntitlement` attribute values in SAML or `eduperson_entitlement`
+claim in OIDC). These entitlements are typically used to indicate access rights
+to protected resources. Entitlements are multi-valued, with each value formatted
+as a URN.
+
+#### Syntax
+
+An entitlement value expressing GOCDB roles has the following syntax
+(components enclosed in square brackets are OPTIONAL):
+
+```vim
+urn:mace:egi.eu:goc.egi.eu:<PRIMARY_KEY>:<ON_ENTITY>:<USER_ROLE>@egi.eu
+```
+
+where:
+
+- `<PRIMARY_KEY>` is ...
```

Comment:
is the primary key for the user role, e.g. "123G0"

Reply:
Done